### PR TITLE
feat(rule): add boolean rule

### PR DESCRIPTION
**Pull request by kkotelczuk**
Added rule for `Boolean` 
Added `chai`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "xo": "^0.17.1"
   },
   "dependencies": {
+    "chai": "^3.5.0",
```
Comment (Idered) [inline package.json:55]: Check https://github.com/Syncano/syncano-client-js/blob/master/tests/index.js to see how to use chai assert

Comment (Idered) [reply package.json:55]: Any progress on this?

Comment (kkotelczuk) [reply package.json:55]: Yes, I'm working on it.

**Review by Idered — APPROVED** [review]
LGTM

**Pull request merged**